cloudferrylib/os/actions/detach_used_volumes_via_compute.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+import copy
+
+from cloudferrylib.base.action import action
+from cloudferrylib.utils import utils as utl
```

Comment:
I really don't see any value in saving 2 bytes on names. Let's simply use `utils` it's more readable, and has no negative impact.